examples/misalignment/role_playing_multiprocess.py

```diff
@@ -43,9 +42,7 @@ def generate_data(
         with_task_specify=True,
         with_task_planner=False,
         task_type=TaskType.MISALIGNMENT,
-        task_specify_agent_kwargs=dict(
-            model_config=ChatGPTConfig(temperature=1.4)
-        ),
+        task_specify_agent_kwargs=dict(),
```

Comment:
The original initial value is `temperature=1.4`, don't we need it anymore?

Reply:
I think it's not important or necessary but nice to have, have added it back